src/info/mod.rs

```diff
@@ -408,13 +406,31 @@ mod tests {
         config.input = repo.path().to_path_buf();
         let info = Info::new(&config)?;
         let info_str = format!("{}", info);
-        let expected_info = fs::read_to_string(repo.path().join("..").join("expected"))?;
-        let escaped_expected_info = regex::escape(&expected_info);
-        let expected_info_expression =
-            escaped_expected_info.replace("MATCH_UNTIL_END_OF_LINE", ".*");
-        let re = Regex::new(format!(r#"{}"#, expected_info_expression).as_str()).unwrap();
-        if !re.is_match(&info_str) {
-            assert_eq!(info_str, expected_info);
+        let info_u8 = strip_ansi_escapes::strip(&info_str)?;
```

Comment:
@spenserblack  and @o2sh 
The function `strip_ansi_escapes::strip` strips the colors and allows a simpler regex which is more readable.

I selected to use a multi-line regex so that the order of fields can be verified in the test.